experimental/CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ All notable changes to experimental packages in this project will be documented
     * `appendResourcePathToUrlIfNeeded`
     * `configureExporterTimeout`
     * `invalidTimeout`
+* Changed `diag.error` to `diag.warn` for unknown `OTEL_NODE_RESOURCE_DETECTORS` values in [#4882](https://github.com/open-telemetry/opentelemetry-js/issues/4882).
```

Comment:
Hi, you will need to change the format here to be the same as the PR title, meaning, including the `fix(sdk-node): ...`, so it's the same format as the other entries here

Reply:
Oh and the link should be to this own PR, so https://github.com/open-telemetry/opentelemetry-js/pull/5034/, and not the issue link

Reply:
Ok @maryliag on it,thank you for the feedback